doc/template.html

```diff
@@ -53,5 +53,6 @@ <h2>Table of Contents</h2>
       </div>
     </div>
   </div>
+  <script src="https://nodejs.org/static/js/doc-patches.js"></script>
```

Comment:
sorry, I have 404 error with this link https://nodejs.org/static/js/doc-patches.js are you sure about the link (I can made error)